webapp/channels/src/components/settings_sidebar/settings_sidebar.tsx

```diff
@@ -110,17 +110,31 @@ export default class SettingsSidebar extends React.PureComponent<Props> {
             pluginTabList = (
                 <>
                     <hr/>
-                    <li
-                        key={'plugin preferences heading'}
-                        role='heading'
-                        className={'header'}
+                    <div
+                        aria-labelledby={'userSettingsModal.pluginPreferences.header'}
+                        role='region'
                     >
-                        <FormattedMessage
+                        <div
                             id={'userSettingsModal.pluginPreferences.header'}
-                            defaultMessage={'PLUGIN PREFERENCES'}
-                        />
-                    </li>
-                    {this.props.pluginTabs.map((tab, index) => this.renderTab(tab, index))}
+                            key={'plugin preferences heading'}
+                            role='heading'
+                            aria-level={3}
```

Comment:
question: how did you determine this is aria-level 3? I ask because since we will be working in several similar changes, so we have clarity how to define those levels.

Reply:
Currently, the setting modal has a heading `Settings` which is an h1, this is wrong it should be h2 and will be fixed in this [issue](https://mattermost.atlassian.net/browse/MM-61684) hence as settings become an h2 then correct hierarchy should be h3 or aria-level=3. I believe the proper way should be to look for its importance and the hierarchy. 
Wave testing extension could help to easily check the page structure to just get how the current hierarchy looks like this helps to get the idea of what should be a more appropriate heading level for an element.

Reply:
The settings modal currently features a heading titled "Settings" marked as an h1, which is incorrect. This issue will be addressed in the linked [issue](https://mattermost.atlassian.net/browse/MM-61684), Once the settings heading becomes h2 then naturally other headings should be h3 or h4 or aria-level 3 or 4. Hence added an aria-level 3 here. This was also recommended in the issue.